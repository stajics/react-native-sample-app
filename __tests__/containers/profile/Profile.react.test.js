import 'react-native';
import React from 'react';
import { Profile } from '../../../app/containers/profile/Profile.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

it('renders correctly', () => {
  const profile = renderer.create(
    <Profile
      text={'TEST TEXT'}
      title={'TEST TITLE'}
    />
  ).toJSON();
  expect(profile).toMatchSnapshot();
});
