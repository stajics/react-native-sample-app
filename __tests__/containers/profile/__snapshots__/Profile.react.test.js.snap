exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 50,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}>
    Im the 
    TEST TEXT
     component with title 
    TEST TITLE
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderColor": "black",
            "borderWidth": 1,
            "height": 20,
            "justifyContent": "center",
            "marginTop": 50,
            "width": 100,
          },
          Object {
            "alignItems": "center",
            "borderColor": "black",
            "borderWidth": 1,
            "height": 35,
            "justifyContent": "center",
            "marginTop": 50,
            "width": 120,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "fontSize": 15,
            },
            Object {
              "fontSize": 20,
            },
          ]
        }>
        SETTINGS
      </Text>
    </View>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderColor": "black",
            "borderWidth": 1,
            "height": 20,
            "justifyContent": "center",
            "marginTop": 50,
            "width": 100,
          },
          Object {
            "alignItems": "center",
            "borderColor": "black",
            "borderWidth": 1,
            "height": 35,
            "justifyContent": "center",
            "marginTop": 50,
            "width": 120,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "fontSize": 15,
            },
            Object {
              "fontSize": 20,
            },
          ]
        }>
        LOGOUT
      </Text>
    </View>
  </View>
</View>
`;
